kotlintest-samples/kotlintest-samples-gradle/build.gradle

```diff
@@ -1,15 +1,5 @@
 apply plugin: 'org.junit.platform.gradle.plugin'
 
-buildscript {
-    ext.kotlin_version = '1.2.31'
-    repositories {
-        mavenCentral()
-    }
-    dependencies {
-        classpath "org.jetbrains.kotlin:kotlin-gradle-plugin:$kotlin_version"
-    }
-}
-
 dependencies {
```

Comment:
Won't this project need the kotlin gradle plugin for the non test dependencies?

Reply:
Good catch. Sorry about that, I didn't realize that the samples weren't submodules.